pages/redirect/humanid.tsx

```diff
@@ -0,0 +1,16 @@
+import getConfig from "next/config"
+
+export async function getServerSideProps({ query, req }) {
+    const { REDIRECT_MAIN_URL } = getConfig().serverRuntimeConfig
+    const queryParamString = new URLSearchParams(query).toString()
+    return {
+        redirect: {
+            destination: `${REDIRECT_MAIN_URL}/verification?${queryParamString}`,
+            permanent: false,
+        },
+    }
+}
+
+export default function HumanIdRedirect() {
+    return <div>Redirecting...</div>
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `HumanIdRedirect` component is simple and does its job of displaying a message while redirection is in progress. However, consider adding more user-friendly information or a loading spinner for better UX.

```diff
export default function HumanIdRedirect() {
    return <div>Redirecting...</div>
+    // Consider adding more user-friendly information or a loading spinner
}
\ No newline at end of file
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->